src/flowr/internal-session.ts

```diff
@@ -21,8 +21,13 @@ export class FlowrInternalSession {
 		this.shell.tryToInjectHomeLibPath()
```

Comment:
This should no longer be necessary with us dropping xmlparsedata

Reply:
It only exists for the statistic components etc. which (still) require libraries